pennylane/qnode.py

```diff
@@ -155,6 +157,10 @@ def __init__(
         **diff_options,
     ):
 
+        if diff_method is None:
+            diff_method = "best"

```

Comment:
Just catching up with the details here, how come we're setting `diff_method = "best"`? Could `diff_method = None` work?

Reply:
As explained in other comments, `diff_method=None` does not exist apart from these lines. Its only job is to transfer workflow to `interface=None`.
Setting `diff_method="best"` because it will be most compatible with whatever the user might try next.
Any error or issue that might happen next was there to begin with and is not a cause of this change.